nixos/modules/services/networking/ntp/chrony.nix

```diff
@@ -218,5 +257,18 @@ in
           SystemCallFilter = [ "~@cpu-emulation @debug @keyring @mount @obsolete @privileged @resources" "@clock" "@setuid" "capset" "@chown" ];
         };
       };
+
+    assertions = [
+      {
+        assertion = cfg.enableRTCTrimming && builtins.any (line: (builtins.match "^ *rtcsync" line) != null) (lib.strings.splitString "\n" cfg.extraConfig);
```

Comment:
I think that it would be better to simply add another option, something like enableRTCSync and add it to this assertion.

Reply:
That wouldn't solve the problem that there was no option like that, users might have enabled it in extraConfig and chrony crashes if rtcsync and rtcfile is both specified.